README.md

```diff
@@ -258,12 +258,29 @@ For a more detailed explanation you can read the full discussion in the [issue #
 
 The Puli integration is completely optional and not required to use this standard. It is only here to facilitate usage with Puli.
 
+Using Puli, packages can declare the service providers they contain. Then, containers can automatically retrieve the list of service providers.
```

Comment:
:+1: in this quick intro, makes more sense than jumping straight into technical details. However it now duplicates this paragraph:

> This way, frameworks or applications based on Puli can discover service providers automatically.

I think it can be removed. And we can even simplify the whole thing (the simpler the better IMO):

> To do so, this package provides a [Puli *binding type*](http://docs.puli.io/en/latest/discovery/getting-started.html): `container-interop/service-provider`. Modules using Puli and implementing this standard can register service providers (fully qualified class names) through this binding type:

> ```php
> puli bind --class Acme\\Foo\\MyServiceProvider container-interop/service-provider
> ```